Instagram/Controllers/Search/SearchViewController.swift

```diff
@@ -0,0 +1,202 @@
+                                           Post(postImageName: Constants.eleganceHasbikImageName),
+                                           Post(postImageName: Constants.hasbikImageName),
+                                           Post(postImageName: Constants.policeHasbikImageName),
+                                           Post(postImageName: Constants.cuteHasbikImageName),
+                                           Post(postImageName: Constants.gangstaHasbikImageName),
+                                           Post(postImageName: Constants.suetologHasbikImageName),
+                                           Post(postImageName: Constants.eleganceHasbikImageName),
+                                           Post(postImageName: Constants.suetologHasbikImageName),
+                                           Post(postImageName: Constants.policeHasbikImageName),
+                                           Post(postImageName: Constants.hasbikImageName)]
+  private let profileDescription = ProfileDescription(profileImageName: Constants.hasbikImageName,
+                                                      postsAmount: Constants.postsAmount,
+                                                      subscribersAmount: Constants.subscribersAmount,
+                                                      subscriptionsAmount: Constants.subscriptionsAmount,
+                                                      profileName: Constants.profileHasbikName,
+                                                      typeOfAccountText: Constants.typeOfProfileName,
+                                                      descriptionText: Constants.descriptionProfileName,
+                                                      sameSubscribesText: Constants.sameSubscribersName)
+  private let profileHighlights = [ProfileHighlight(highlightImageName: Constants.hasbikImageName,
+                                               highlightName: Constants.firstHighlightName),
+                              ProfileHighlight(highlightImageName: Constants.policeHasbikImageName,
+                                               highlightName: Constants.secondHighlightName),
+                              ProfileHighlight(highlightImageName: Constants.gangstaHasbikImageName,
+                                               highlightName: Constants.thirdHighlightName),
+                              ProfileHighlight(highlightImageName: Constants.suetologHasbikImageName,
+                                               highlightName: Constants.forthHighlightName),
+                              ProfileHighlight(highlightImageName: Constants.cuteHasbikImageName,
+                                               highlightName: Constants.fithHighlightName),
+                              ProfileHighlight(highlightImageName: Constants.eleganceHasbikImageName,
+                                               highlightName: Constants.sixthHighlightName)]
+  // MARK: - Life cycle.
```

Comment:
```suggestion

  // MARK: - Life cycle.
```
отступы везде после методов